server/renderer.js

```diff
@@ -193,12 +193,14 @@ async function serverRenderAuthorised(req, res, next, htmlData, uiLang, bot) {
               <React.StrictMode>
                 <ErrorBoundary>
                   <HelmetProvider context={helmetContext}>
-                    <App
-                      i18n={context.i18n}
-                      store={store}
-                      history={history}
-                      deviceInfo={deviceInfo}
-                    />
+										<Suspense fallback="">
```

Comment:
As I understand for use Suspense  you need change SSR - for now its return html as string but need as stream. 
@bbfsdev